### PR TITLE
sys-block/arcconf: bump to EAPI=7

### PR DESCRIPTION
**Pull request by ConiKost**
Also bumped to 2.06.23168.

Closes: https://bugs.gentoo.org/669776
Signed-off-by: Conrad Kostecki <conrad@kostecki.com>
Package-Manager: Portage-2.3.51, Repoman-2.3.11

### CODE REVIEW

File: sys-block/arcconf/arcconf-2.06.23167.ebuild [hunk 1]
```diff
@@ -1,7 +1,7 @@
-# Copyright 1999-2018 Gentoo Foundation
+# Copyright 1999-2018 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=7
+EAPI="7"
```
Comment (juippis) [inline sys-block/arcconf/arcconf-2.06.23167.ebuild]: Is this change really necessary?

Comment (ConiKost) [reply sys-block/arcconf/arcconf-2.06.23167.ebuild]: No, just did it, since all my other eBuilds have it this way ;-)

Comment (juippis) [reply sys-block/arcconf/arcconf-2.06.23167.ebuild]: >since all my other eBuilds have it this way

Some devs strip the quotation marks off when pushing your commit to gentoo tree (if you weren't aware), so isn't this like fighting a windmill? :)

File: sys-block/arcconf/arcconf-2.01.22270-r1.ebuild [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+DESCRIPTION="Microsemi Adaptec RAID Controller Command Line Utility"
+HOMEPAGE="https://www.microsemi.com/"
+SRC_URI="https://download.adaptec.com/raid/storage_manager/${PN}_${MY_PV}.zip"
+
+KEYWORDS="-* ~amd64 ~x86"
+LICENSE="Microsemi"
+SLOT="0"
+
+DEPEND="app-arch/unzip"
+RDEPEND="${RDEPEND}"
```
Comment (a17r) [inline sys-block/arcconf/arcconf-2.01.22270-r1.ebuild]: Sure `app-arch/unzip` is needed at runtime?

File: sys-block/arcconf/arcconf-2.06.23167.ebuild [hunk 3]
```diff
@@ -1,4 +1,4 @@
-# Copyright 1999-2018 Gentoo Foundation
```
Comment (a17r) [inline sys-block/arcconf/arcconf-2.06.23167.ebuild]: why touch that ebuild for Copyright only?

File: sys-block/arcconf/metadata.xml [hunk 4]
```diff
@@ -1,10 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
 <pkgmetadata>
-	<maintainer type="person">
-		<email>dev-zero@gentoo.org</email>
```
Comment (a17r) [inline sys-block/arcconf/metadata.xml]: Is this approved by @dev-zero?

Comment (ConiKost) [reply sys-block/arcconf/metadata.xml]: Yes, see the git commit, it's written there.

File: sys-block/arcconf/arcconf-2.06.23167.ebuild (hunk 3 above)
Comment (ConiKost) [reply sys-block/arcconf/arcconf-2.06.23167.ebuild]: That was a misstake. Sorry.

File: sys-block/arcconf/arcconf-2.01.22270-r1.ebuild (hunk 2 above)
Comment (ConiKost) [reply sys-block/arcconf/arcconf-2.01.22270-r1.ebuild]: Nope, that was a misstake. Removed.

**Comment by ConiKost** [comment]
Also bumped now to 2.06.23168.
